zcash_client_sqlite/src/wallet/scanning.rs

```diff
@@ -1322,14 +1364,26 @@ mod tests {
                     // complete the left-hand side of the required shard; this can be fixed once we
                     // have proper account birthdays.
```

Comment:
The comment above should now be out-of-date, given that we now have proper account birthdays? Can this test be fixed?

Reply:
The comment here is just wrong; we don't actually scan from below the account birthday in the test below. Removing.